assets/js/script.js

```diff
@@ -0,0 +1,291 @@
+  var $playerTwo = $('.playerTwo')
+  $('.playerOneDiv').addClass("player1")
+  $('.playerTwoDiv').addClass("player2")
+  $('.instructorDiv').addClass("instructorPic")
+  var $resetButton = $('<button class = "resetBtn">Reset</button>')
+  var randSequence = shuffleArray(instructorMoves)
+
+
+  //Part 2a - Add turns after round 1
+  function addTurn(){
+    instructorMoves = instructorMoves.concat(instructorMoves)
+    randSequence = shuffleArray(instructorMoves)
+    return randSequence;
+  }
+  //Making the instructor move according to number sequence
+  function instructorDance () {
+    var instructorMovesPic = [
+      { number: 37, image: 'url("./assets/images/instructor_1.png")' },
+      { number: 38, image: 'url("./assets/images/instructor_2.png")' },
+      { number: 39, image: 'url("./assets/images/instructor_3.png")' },
+      { number: 40, image: 'url("./assets/images/instructor_4.png")' }
+    ];
+    $('.instructorDiv').addClass("move");
+    var timeoutInterval = 1000
+    let test = randSequence.map(function(value) {
+      return (value - 36);
+    })
+
+    for(let e = 0; e <= randSequence.length; e++){
+      for(let i = 0; i < instructorMovesPic.length; i++) {
+        if(instructorMovesPic[i].number === randSequence[e]){
+          setTimeout(function() {
+            $('.instructorDiv').css('backgroundImage', instructorMovesPic[i].image);
+            $instructor.text(test.slice(0,e+1))
+          }, timeoutInterval * (e*0.4))
+        }
+      }
+    }
+  }
+
+  //Adding timer, execute addTurn and call instructorDance
+  function timerWithAddRounds (){
+    var timer = 6;
+    var timerInverval = setInterval(function() {
+      timer--;
+      if(timer >= 0){
+        return $instructor.text(timer)
+      } else {
+        clearInterval(timerInverval)
+        if (rounds > 1) addTurn()
+        instructorDance()
+        return  instructorDance()
+      }
+    }, 1000);
+  }
+
+  $('.resetBtn').on('click', function(){
+  document.location.reload(true);
+  })
+
+  //Looping my background music
+  myAudio = new Audio('./assets/audio/bgm.mp3')
+  myAudio.addEventListener('ended', function() {
+  this.currentTime = 0;
+  this.play();
+  }, false);
+
+  $button.on('click' , function () {
+    myAudio.play();
+    timerWithAddRounds()
+    $(".startBtn").hide()
+    $(".resetBtn").show()
+
+  $body.on('keydown', (e) => {
+
+    if(e.which === 49 || e.which === 50|| e.which === 51|| e.which === 52){
+      whichPlayer = 1
+    }
+    if(e.which === 37 || e.which === 38|| e.which === 39|| e.which === 40){
+      whichPlayer = 2
+    }
+
+    //Part 3 - Matching keypressed with instructorMoves in sequence
+    var strOfKeys = e.which - 12 //Player one's key stroke
+    var strOfKeys1 = e.which    //Player two's key stroke
+
+    // Set keys with different images for player 1
+    if(strOfKeys === 37) {
+    $('.playerOneDiv').addClass("move");
+    $('.playerOneDiv').css('backgroundImage', `url("./assets/images/player1_1.png")`);
```

Comment:
again same issues here like the instructor moves, why just add in the class instead?